pkgdb2/api/packages.py

```diff
@@ -1287,9 +1291,11 @@ def api_package_request():
             cnt += 1
         collections.append(collection)
 
+    namespaces = pkgdblib.get_status(SESSION, 'namespaces')['namespaces']
```

Comment:
It's the idea yes (getting all of them from the DB to populate the drop-down list)